library/CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ set(src_crypto
     md4.c
     md5.c
     memory_buffer_alloc.c
+    mps/reader.o
```

Comment:
Thanks for spotting that. Because of the final number of MPS source files, I'm a bit reluctant to inline them into `library/` and would much prefer an `mps` subdirectory. I'll take a look at how to _properly_ do this with CMake.